Version_testNet

```diff
@@ -1,3 +1,3 @@
 DOCKER_IMAGE_FUSE_APP_VERSION="1.0.0"
-DOCKER_IMAGE_FUSE_PARITY_VERSION="1.0.0_OE"
+DOCKER_IMAGE_FUSE_PARITY_VERSION="2.0.0_OE"
```

Comment:
this called PARITY but this is OE ><

Reply:
yeah more to keep backwards compatibility here so you can still update containers by rerunning older quickstarts (using the version file)